### PR TITLE
AbstractFactory

### PR DESCRIPTION
**Pull request by Stanislava88**
Here i have solved the task for the creation of House Part objects using pattern named AbstractFactory

### CODE REVIEW

File: src/com/clouway/abstractfactory/Window.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
+package com.clouway.abstractfactory;
+
+/**
+ * The implementation of this interface will be used to get color to window
```
Comment (freekman) [inline src/com/clouway/abstractfactory/Window.java]: ....  used to get window information

File: src/com/clouway/abstractfactory/WindowFactory.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
+package com.clouway.abstractfactory;
+
+/**
+ * @author Stanislava Kaukova(sisiivanovva@gmail.com)
+ */
+public class WindowFactory extends HousePartFactory {
+    private static String type1 = "wooden";
+    private static String type2 = "aluminium";
```
Comment (freekman) [inline src/com/clouway/abstractfactory/WindowFactory.java]: please remove the aluminium implementation for floor ,window .... use only the wood imp the PR is 400 line for a patttern

File: src/com/clouway/abstractfactory/WindowFactory.java [hunk 3]
```diff
@@ -0,0 +1,30 @@
+package com.clouway.abstractfactory;
+
+/**
+ * @author Stanislava Kaukova(sisiivanovva@gmail.com)
+ */
+public class WindowFactory extends HousePartFactory {
+    private static String type1 = "wooden";
```
Comment (freekman) [inline src/com/clouway/abstractfactory/WindowFactory.java]: names like tipe1 , obj1, ..... give you no value use ```wood ```

Comment (freekman) [inline src/com/clouway/abstractfactory/WindowFactory.java]: why static

File: src/com/clouway/abstractfactory/Window.java (hunk 1 above)
Comment (Stanislava88) [inline src/com/clouway/abstractfactory/Window.java]: done

**Comment by freekman** [comment]
Please make both PR for Abstract Factory and Factory method to work with one imp only wood you are complicating too much

File: src/com/clouway/abstractfactory/WindowFactory.java (hunk 2 above)
Comment (Stanislava88) [inline src/com/clouway/abstractfactory/WindowFactory.java]: done

**Comment by Stanislava88** [comment]
done

**Comment by freekman** [comment]
LGTM

**Pull request merged**